pkg/controller/status/status.go

```diff
@@ -160,7 +171,8 @@ func (m *statusManager) OnCRNotFound() {
 	m.clearProgressing()
 	m.lock.Lock()
 	defer m.lock.Unlock()
-	m.enabled = false
+	_f := false
```

Comment:
what's the `_` for? Is that a convention I'm not aware of?

Reply:
I thought I've seen that before for variables that are only created for the purpose of being able to get a pointer to them. If not I'll drop the `_`.